src/HOI4World/PeaceConferences/IdeologicalAiPeaceTests.cpp

```diff
@@ -0,0 +1,24 @@
+#include "external/googletest/googletest/include/gtest/gtest.h"
+#include "src/HOI4World/PeaceConferences/IdeologicalAiPeace.h"
+#include "gmock/gmock-matchers.h"
+
+
+
+TEST(HoI4World_PeaceConferences_IdeologicalAiPeaceTests, ItemsCanBeImported)
+{
+	const auto ideological_ai_peace = HoI4::ImportIdeologicalAiPeace();
+
+	EXPECT_THAT(ideological_ai_peace,
+		 testing::UnorderedElementsAre(testing::Pair("test_ideology",
+													  "\tblock_one = {\n"
```

Comment:
I think there's a ton of whitespace trailing these blocks. Just cosmetics.

Reply:
Unfortunately that's what the formatter likes, strange as it is.